package.json

```diff
@@ -87,28 +84,23 @@
     "@babel/core": "^7.5.5",
     "@babel/preset-env": "^7.5.5",
     "@babel/preset-react": "^7.0.0",
-    "@types/mapbox-gl": "^0.54.2",
```

Comment:
This is needed for auto-completion in my IDE, so I would keep it / upgrade it to 54.3 when we upgraded mapbox-gl to 1.3.0

Reply:
Let's put it in `optionalDependencies`?